tools/functional-tester/docker/docker-compose.yml

```diff
@@ -19,10 +19,5 @@ tester:
     - /etcd-tester
     - -agent-endpoints
     - "172.20.0.2:9027,172.20.0.3:9027,172.20.0.4:9027"
-    - -limit 
```

Comment:
should the functional-tester run on docker image mirrors the one we run using goreman?